appveyor.yml

```diff
@@ -34,12 +34,13 @@ before_deploy:
   - move /Y %APPVEYOR_BUILD_FOLDER%\packaging\windows\OpenRA.Setup.exe %APPVEYOR_BUILD_FOLDER%\OpenRA-%APPVEYOR_REPO_TAG_NAME%.exe
 
 artifacts:
-  - path: OpenRA-%APPVEYOR_REPO_TAG_NAME%.exe
+  - name: Installer
+    path: OpenRA-%APPVEYOR_REPO_TAG_NAME%.exe

```

Comment:
And another thing I just saw: it seems that the syntax for env variables in that place is $(envVarName), not %envVarName%. (see the third paragraph on http://www.appveyor.com/docs/packaging-artifacts)